owncloudApp/src/test-common/java/com/owncloud/android/utils/TestUtil.kt

```diff
@@ -119,6 +124,146 @@ object TestUtil {
+        sharingPublicPasswordEnforced: Int = 0,
+        sharingPublicPasswordEnforcedReadOnly: Int = 0,
+        sharingPublicPasswordEnforcedReadWrite: Int = 0,
+        sharingPublicPasswordEnforcedUploadOnly: Int = 0,
+        sharingPublicExpireDateEnabled: Int = 0,
+        sharingPublicExpireDateDays: Int = 0,
+        sharingPublicExpireDateEnforced: Int = 0,
+        sharingPublicSendMail: Int = 0,
+        sharingPublicUpload: Int = 0,
+        sharingPublicMultiple: Int = 0,
+        sharingPublicSupportsUploadOnly: Int = 0,
+        sharingUserSendMail: Int = 0,
+        sharingResharing: Int = 0,
+        sharingFederationOutgoing: Int = 0,
+        sharingFederationIncoming: Int = 0,
+        filesBigFileChunking: Int = 0,
+        filesUndelete: Int = 0,
+        filesVersioning: Int = 0
+    ) = OCCapability(
+        accountName,
+        versionMayor,
+        versionMinor,
+        versionMicro,
+        versionString,
+        versionEdition,
+        corePollinterval,
+        sharingApiEnabled,
+        sharingPublicEnabled,
+        sharingPublicPasswordEnforced,
+        sharingPublicPasswordEnforcedReadOnly,
+        sharingPublicPasswordEnforcedReadWrite,
+        sharingPublicPasswordEnforcedUploadOnly,
+        sharingPublicExpireDateEnabled,
+        sharingPublicExpireDateDays,
+        sharingPublicExpireDateEnforced,
+        sharingPublicSendMail,
+        sharingPublicUpload,
+        sharingPublicMultiple,
+        sharingPublicSupportsUploadOnly,
+        sharingUserSendMail,
+        sharingResharing,
+        sharingFederationOutgoing,
+        sharingFederationIncoming,
+        filesBigFileChunking,
+        filesUndelete,
+        filesVersioning
+    )
+
+    fun createRemoteCapability(
+        accountName: String = "user@server",
+        versionMayor: Int = 2,
+        versionMinor: Int = 1,
+        versionMicro: Int = 0,
+        versionString: String = "1.0.0",
+        versionEdition: String = "1.0.0",
+        corePollinterval: Int = 0,
+        sharingApiEnabled: Int = 0,
+        sharingPublicEnabled: Int = 1,
+        sharingPublicPasswordEnforced: Int = 0,
+        sharingPublicPasswordEnforcedReadOnly: Int = 0,
+        sharingPublicPasswordEnforcedReadWrite: Int = 0,
+        sharingPublicPasswordEnforcedUploadOnly: Int = 0,
+        sharingPublicExpireDateEnabled: Int = 0,
+        sharingPublicExpireDateDays: Int = 0,
+        sharingPublicExpireDateEnforced: Int = 0,
+        sharingPublicSendMail: Int = 0,
+        sharingPublicUpload: Int = 0,
+        sharingPublicMultiple: Int = 0,
+        sharingPublicSupportsUploadOnly: Int = 0,
+        sharingUserSendMail: Int = 0,
+        sharingResharing: Int = 0,
+        sharingFederationOutgoing: Int = 0,
+        sharingFederationIncoming: Int = 0,
+        filesBigFileChunking: Int = 0,
+        filesUndelete: Int = 0,
+        filesVersioning: Int = 0
+    ): RemoteCapability {
+        val remoteCapability = RemoteCapability()
+
```

Comment:
Could we rewrite this using apply?

`): RemoteCapability = RemoteCapability().apply{
 ... 
}`